common/internal_model/src/main/scala/weco/catalogue/internal_model/index/WorksAnalysis.scala

```diff
@@ -78,12 +87,31 @@ object WorksAnalysis {
     charFilters = Nil
   )
 
+  // now also include a case sensitive version of the english analyzer
+  val englishCasedAnalyzer = CustomAnalyzer(
+    "english_cased_analyzer",
+    tokenizer = "standard",
+    tokenFilters = List(
+      asciiFoldingTokenFilter.name,
+      englishStemmerTokenFilter.name,
+      englishPossessiveStemmerTokenFilter.name
+    ),
+    charFilters = Nil
+  )
+
   val shingleAsciifoldingAnalyzer = CustomAnalyzer(
     "shingle_asciifolding_analyzer",
     tokenizer = "standard",
     tokenFilters =
       List("lowercase", shingleTokenFilter.name, asciiFoldingTokenFilter.name),
-    charFilters = Nil
+    charFilters = List(hyphensCharFilter.name)
+  )
+
+  val shingleCasedAnalyzer = CustomAnalyzer(
+    "shingle_cased_analyzer",
+    tokenizer = "standard",
+    tokenFilters = List(shingleTokenFilter.name, asciiFoldingTokenFilter.name),
+    charFilters = List(hyphensCharFilter.name)
   )
```

Comment:
What does this analyser do?

Reply:
In elastic land, shingles are spans of multiple words which are indexed as one token, while preserving their order. As a simple example, a 3-gram shingles analyser would index "the quick brown fox jumped over the lazy dog" as 
- "the quick brown"
- "quick brown fox"
- "brown fox jumped"
- "fox jumped over"
- "jumped over the"
- "over the lazy"
- "the lazy dog"

The existing `shingleAsciifoldingAnalyzer` lowercases and asciifolds all of those tokens (and the queries which it compares them to). This new `shingleCasedAnalyzer` does the same thing, but doesn't apply a lowercase token filter. That should make us even more likely to retrieve strings of text where both casing and position are important, eg when people search for "hearing aids", a work called "hearing the stories of AIDS survivors" should appear lower in the results than "a story about hearing aids".